lib/Runtime/Library/JavascriptArray.cpp

```diff
@@ -8648,9 +8652,8 @@ using namespace Js;
     }
 
     ///----------------------------------------------------------------------------
-    /// FindIndex() calls the given predicate callback on each element of the
-    /// array, in order, and returns the index of the first element that makes the
-    /// predicate return true, as described in (ES6.0: S22.1.3.9).
+    /// FindLast() calls the given predicate callback on each element of the array, in
```

Comment:
I think this comment is in the wrong - it's describing FindLast but is above FindIndex